Glossario.json

```diff
@@ -1,322 +1,322 @@
+		"acronyms": [],
+		"synonyms": ["Registro delle modifiche"]
+	},
+	"Commit": {
+		"description": "Nei Version Control System rappresenta la registrazione all'interno di una repository di una serie di modifiche apportate ad un documento o a del codice sorgente. Ogni commit è dotato di un codice identificativo, e contribuisce a creare, nel complesso, la cronologia del progetto.",
+		"acronyms": [],
+		"synonyms": []
+	},
+	"Committente": {
+		"description": "Figura che commissiona un lavoro. Può essere una persona fisica oppure una persona giuridica (esempio un ente o un'azienda).",
+		"acronyms": [],
+		"synonyms": []
+	},
+	"Database": {
+		"description": "Insieme di informazioni (o dati) strutturate, in genere archiviate elettronicamente in un sistema informatico. Solitamente i database sono gestiti da un DBMS (Database Management System), il quale deve garantire efficacia, efficienza, privacy ed affidabilità.",
+		"acronyms": ["DB"],
+		"synonyms": []
+	},
+	"Discord": {
+		"description": "Piattaforma di VoIP (Voice over IP, ovvero \"Voce tramite protocollo Internet\"), messaggistica istantanea e distribuzione digitale. Gli utenti possono comunicare tramite chat private, gruppi o come membri di uno stesso server, ovvero ambienti virtuali suddivisi in canali di testo o vocali.",
+		"acronyms": [],
+		"synonyms": []
+	},
+	"Docker": {
+		"description": "Software progettato per semplificare il processo di deployment di applicazioni software. Permette di eseguire processi informatici in ambienti isolati chiamati container.",
+		"acronyms": [],
+		"synonyms": []
+	},
+	"Documentazione": {
+		"description": "È un testo scritto allo scopo di spiegare quali funzionalità assolve il software, come è strutturato, implementato e come lo si utilizza.",
+		"acronyms": [],
+		"synonyms": []
+	},
+	"Feature": {
+		"description": "Specifica funzionalità o caratteristica all'interno del prodotto sviluppato.",
+		"acronyms": [],
+		"synonyms": []
+	},
+	"Git": {
+		"description": "Software di versionamento distribuito utilizzabile tramite linea di comando.",
+		"acronyms": [],
+		"synonyms": []
+	},
+	"GitHub": {
+		"description": "Servizio di hosting di progetti software basato su Git utilizzato da singoli ed organizzazioni. Fornisce un Issue Tracking System integrato e permette di implementare sistemi di Continuous Integration e Continuous Delivery tramite automazioni chiamate Actions.",
+		"acronyms": ["GH"],
+		"synonyms": []
+	},
+	"GitHub Actions": {
+		"description": "Piattaforma integrata di GitHub per la Continuous Integration e la Continuous Delivery che consente ai team di sviluppo di automatizzare il flusso di lavoro, comprese le pipeline di compilazione, test e distribuzione.",
+		"acronyms": ["GHA"],
+		"synonyms": []
+	},
+	"GitHub Pages": {
+		"description": "Piattaforma offerta da GitHub, utilizzata per ospitare siti web statici che si basano su repository.",
```

Comment:
cambierei in:
"Servizio di hosting web fornito da GitHub, progettato per ospitare siti web statici generati da file presenti in repository."